src/krkn_lib/models/telemetry/models.py

```diff
@@ -11,63 +11,63 @@ class ScenarioTelemetry:
     Scenario Telemetry collection
     """
 
-    startTimeStamp: float
+    start_timestamp: float
```

Comment:
since we started I mixed snake case with camel case and I've the feeling that this sounds ugly. I'd suggest to fix this problem.

Reply:
sounds good to me, I **try** to use the underscores/prefer them